FinniversKit/Sources/SwiftUI Components/Button/SwiftUIIconButton.swift

```diff
@@ -2,8 +2,8 @@ import SwiftUI
 
 public struct SwiftUIIconButton: View {
     public struct Style {
-        let icon: ImageAsset
-        let iconToggled: ImageAsset
+        let icon: UIImage
```

Comment:
I found it necessary to change this style lets to use `UIImage` instead of `ImageAsset`.  Not sure if this is ideal as ImageAssets are commonly used to render images. I could only find the `.favourites` style used with this component where we actually need brand images. 

I could not find any use of this component where the public extension SwiftUIIconButton.Style was not used, so I dont see any breaking changes as a result of the let change 🤔

Reply:
The only usage of this struct seems to be the property `favorite` on line 26 in this file, so we're good :) Also, the properties are internal so this is not a breaking change 👌